README-CONAN.md

```diff
@@ -289,6 +289,9 @@ If you wish to use the static C run-time library, use the following option in th
 |:---                  |:---------           |:-------------------|
 | **profile setting**  | compiler.runtime=MT | compiler.runtime=MTd |
 
+If you receive a linker warning concerning LIBCMT, it is because you are attempting to link libraries which have been built with different run-time libraries.
```

Comment:
```suggestion
If you receive a linker warning concerning `LIBCMT`, it is because you are attempting to link libraries which have been built with different run-time libraries.
```